### DISCUSSION TITLE
Dawn Theme as base

### DISCUSSION THREAD

**Discussion by mrpacman101** [hayes0724/shopify-packer#117] [Ideas]
@hayes0724 
https://github.com/Shopify/dawn

It would be amazing to get a deeper integration with this theme for packer.

**Comment by qedric** [comment]
It would be very useful to get Shopify CLI integrated, because the existing themekit doesn't recognise the new JSON templates

**Comment by GuillermoCasanova** [comment]
Agree! 

With Dawn there seems to be a few breaking changes, like no more scss. Is there plans to make Packer compile into Shopify Dawn 2.0 friendly structure?  A shame that if we want to use Dawn we have to nix using NPM  and SCSS :( -- Shopify mentions briefly in their docs how "one could implement transformations" but isn't very clear as to how to do this while developing. 

I'd love to compile into Dawn structure but still be able to use NPM and SCSS. For now I'll be making new stores for clients in Dawn with Shopify's CLI as-is. 

If anyone has answers, please share!

**Reply by foamingmonkey** [reply]
my thought is that you can call Shopify API within Node [shopify api](https://www.npmjs.com/package/shopify-api-node) and within that, you can set up something like parcel
